src/functions-and-arrays.js

```diff
@@ -1,15 +1,44 @@
 // Iteration #1: Find the maximum
-
+const maxOfTwoNumbers = (num1, num2) => {
+  if (num1 >num2) {
+    return num1;
+  }else return num2;
+};
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearlllll', 'orchard', 'crackpot'];
+let findLongestWord = (array) => {
+    let word ="";
+    for (let i=0 ; i<array.length; i++ ){
+        if(word.length < array[i].length){
+            word =array[i];
+        }
+    }
+    return word;
+}
```

Comment:
It seems the Jasmine tests for this function are failing with the following warning 👉🏾  `Find the longest word should return null when called with an empty array`

The test expects there to be some handling for empty arrays. Specifically, it wants the function to return `null` if an empty array is passed.
![Screenshot 2021-03-24 at 12 40 49](https://user-images.githubusercontent.com/6001283/112304813-37927a80-8c9e-11eb-826f-56dfb92f04ad.png)


---
One thing you can do here is to add the empty array handling before any of the function's operations. This way, you ensure that you are taking care of this edge case before anything else.

```suggestion
let findLongestWord = (array) => {
/* Handle empty array edge case here:*/
 // 1. Check if the incoming array is empty.
 // 2. Return "null" if it is.
 
// code goes here...

/* Function's operation if the incoming array is not empty */ 

    let word ="";
    for (let i=0 ; i<array.length; i++ ){
        if(word.length < array[i].length){
            word =array[i];
        }
    }
    return word;
}
```